tools/propose-version.mjs

```
@@ -0,0 +1,113 @@
+--lua-source OO2ewZKq4AHoqGQmYUIl-NhJ-llQyFJ3ha4Uf4-w5RI
+
+*/
+
+function getLatestChanges() {
+  const content = fs.readFileSync(path.join(__dirname, '../CHANGELOG.md'), {
+    encoding: 'utf-8',
+  });
+
+  const match = content.match(/## \[\d+\].*?\n(.*?)(?=\n## \[\d+\])/s);
+
+  if (match && match[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)